### DISCUSSION TITLE
Rendering .Rmd with Quarto

### DISCUSSION THREAD

**Discussion by jleclai2** [quarto-dev/quarto-cli#10735] [Q&A] [answered]
### Description

Hello, on the Quarto website, it says "Quarto can render most existing Rmd files without modification."  Can someone please explain why this would be useful?  I was able to use the quarto_render() function to render a Rmd file, but what advantage does this have over just rendering with r markdown?  Thanks!

**Comment by mcanouil** [comment, marked as answer]
You don't need to start R. You can render from the command line using Quarto CLI.
The `quarto` R package is only a wrapper that uses Quarto CLI.
